storage/tianmu/util/bitset.h

```diff
@@ -32,6 +33,7 @@ class BitSet {
       ptr_ = new char[(sz + NO_OF_BITS - 1) / NO_OF_BITS]();
       allocated_ = true;
     }
+    memset(ptr_, 0, (sz + NO_OF_BITS - 1) / NO_OF_BITS);
```

Comment:
You will reset the `data`, which it maybe already stored some data in.

Reply:
Yes, it is. I think how to change it

Reply:
> Yes, it is. I think how to change it

What's your solution?

Reply:
The reason for the problem is that when the update and update data are merged, there is a place where the update bitmap is not set, resulting in the subsequent data reading confusion